test/test_desparsified_lasso.py

```diff
@@ -72,32 +68,31 @@ def test_desparsified_group_lasso():
 
     n_samples = 50
     n_features = 100
-    n_times = 10
+    n_target = 10
     support_size = 2
-    sigma = 0.1
-    rho = 0.9
-    corr = toeplitz(np.geomspace(1, rho ** (n_times - 1), n_times))
-    cov = np.outer(sigma, sigma) * corr
+    signal_noise_ratio = 5000
+    rho_serial = 0.9
```

Comment:
Why are we picking such extreme values for both `rho_serial` and `snr` ?
If there us no good reason I suggest to open another issue and address that later.